### PR TITLE
Update release.md

### PR DESCRIPTION
**Pull request by simrangupta234**
Broken images are corrected.

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
> 
> This content is auto-generated and links may not function. The source of the document is located [here](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/release.md).

I'm afraid to say that if we want to fix this, we should fix the import tool to also fetch images.

/hold
because this is the wrong fix.

File: content/en/releases/release.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ The general labeling process should be consistent across artifact types.
 
 ## The Release Cycle
 
-![Image of one Kubernetes release cycle](release-cycle.png)
+![Image of one Kubernetes release cycle](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/release-cycle.png)
 
 Kubernetes releases currently happen approximately four times per year.
```
Comment (RinkiyaKeDad) [inline content/en/releases/release.md:141]: Also won't we need to change this to `three times per year`? (the image linked also needs to be updated to reflect that there are now three releases in a year).

Comment (sftim) [reply content/en/releases/release.md:141]: Would you be willing to update https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/release.md and then copy that change here using the automation that's in place?

We prefer to make fixes upstream.

Comment (RinkiyaKeDad) [reply content/en/releases/release.md:141]: Sure, will update here once I've made the change on k/community :)

**Comment by reylejano** [comment]
Closing this PR as this is the incorrect fix 
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/release.md should be updated and automation will make the changes to the kubernetes.io site
Issue https://github.com/kubernetes/community/issues/6178 tracks release cycle changes

/close

**Pull request closed without merging**